### ISSUE TITLE
Dynamic reflection class

### ISSUE THREAD

**Issue by SpaceWhite** [github/securitylab#33]
## CVE ID(s)

None

## Report

Created query for unsafe reflection in java https://github.com/Semmle/ql/pull/2756

- [ ] Are you planning to discuss this vulnerability submission publicly? (Blog Post, social networks, etc). *We would love to have you spread the word about the good work you are doing*

**Comment by SpaceWhite** [comment]
Query rejected because of too high false positive rate

**Comment by xcorail** [comment]
Hi @SpaceWhite 

Thanks for the submission! We have reviewed your report and validated your findings. After internally assessing the findings and the query we have determined this query will have a too high false positive rate to make it into our standard query suite, and not eligible for a reward under the Bug Bounty program. We only award higher quality queries as part of the program with attribution on our Bug Bounty site and a payout between $500 and $3,000.

Even though this issue was assessed as being ineligible, we appreciate you bringing it to our attention. We will merge it into the public Security Lab repo as part of the community queries here, and would still like to offer you $200, to encourage you to write more queries.

Best regards and happy hacking!
Xavier, for the GitHub Security Lab
